### PR TITLE
Add support for licensed RDS instances (Oracle, MS-SQL)

### PR DESCRIPTION
**Pull request by wjwoodson**
Allow cf brokered RDS instances to utilize {license-included,bring-your-own-license,general-public-license} licensing models in aws API

### CODE REVIEW

File: catalog-test.yml [hunk 1]
```diff
@@ -158,3 +160,28 @@ rds:
...
+        costs:
+          -
+            amount:
+              usd: 19.19
+            unit: "HOURLY"
+        displayName: "Dedicated Medium Oracle"
+      free: false
+      adapter: dedicated
+      instanceClass: db.m3.medium
+      dbType: oracle
```
Comment (jmcarp) [inline catalog-test.yml]: I'm guessing the sdk will complain: https://github.com/aws/aws-sdk-go/blob/master/service/rds/api.go#L11885-L11916

File: catalog-template.yml [hunk 2]
```diff
@@ -12,6 +12,8 @@ rds:
     - "RDS"
     - "postgresql"
     - "mysql"
+    - "oracle-se1"
```
Comment (cnelson) [inline catalog-template.yml]: Why offer SE1 at all? It's as EOL as anything Oracle ships gets (Sustaining Support only)

Comment (wjwoodson) [inline catalog-template.yml]: oracle-se2 with `license-included` is currently unavailable in GovCloud. We may want to move this to blocked/waiting.

Comment (cnelson) [inline catalog-template.yml]: If SE1 is all that's on offer for license-included, then :shipit:

File: services/rds/rdsinstance.go [hunk 3]
```diff
@@ -32,14 +32,26 @@ type RDSInstance struct {
...
 	re, _ := regexp.Compile("(i?)[^a-z0-9]")
 	return re.ReplaceAllString(i.Database, "")
 }
 
+func (i *RDSInstance) getDbName(DbNamePrefix, RandStr string) string {
+	
+    name := DbNamePrefix + RandStr
+	
+    if i.DbType == "oracle-se1" {
+		return name[0:7]
```
Comment (cnelson) [inline services/rds/rdsinstance.go]: What happens if the prefix is longer than 8 characters?

Comment (jmcarp) [inline services/rds/rdsinstance.go]: Funny story: it is. https://github.com/18F/cg-provision/blob/master/terraform/modules/iam_user/aws_broker_user/policy.json#L23-L29

Comment (jmcarp) [inline services/rds/rdsinstance.go]: Maybe it's time to separate the `DBInstanceIdentifier` parameter from the `DBName` parameter. Looks like the first is used for aws permissions and the second is the thing that's tripping us up. Do we need the prefix to be included in `DBName` at all?

File: ci/pipeline.yml [hunk 4]
```diff
@@ -124,33 +124,44 @@ jobs:
...
-      file: aws-broker-app/ci/run-smoke-tests.yml
-      params:
-        <<: *staging-cf-creds
-        SERVICE_PLAN: medium-psql
-        DB_TYPE: postgres
+    - get: sqlclient-oracle-basiclite
+    - get: sqlclient-oracle-sqlplus
+    - get: sqlclient-postgres
+    - get: sqlclient-mysql
+  - task: smoke-tests-postgres
```
Comment (jmcarp) [inline ci/pipeline.yml]: @wjwoodson: looks like this changes smoke tests to run in series instead of in parallel.

**Comment by rogeruiz** [comment]
Pointing this back to master. but this should now be ready for review and merge.

**Comment by rogeruiz** [comment]
Documentation bits that push us over the AA side of the board. :eyes 18F/cg-site#947

**Pull request merged**